rust/src/mqtt/mqtt.rs

```diff
@@ -436,6 +436,7 @@ impl MQTTState {
                         current,
                         (current.len() - rem.len()) as i64,
                         MQTTFrameType::Pdu as u8,
+                        None,
```

Comment:
@satta should we set tx_id for MQTT frames ?

Reply:
What are the implications of this? (Not sure I understand the "frame" concept good enough)... if it's just adding the transaction ID to the output, then sure, why not?

Reply:
> What are the implications of this? (Not sure I understand the "frame" concept good enough)... if it's just adding the transaction ID to the output, then sure, why not?

Transaction ID, and more important app-layer metadata for this transaction ;-)

Reply:
I think I have to read https://docs.suricata.io/en/latest/devguide/extending/app-layer/app-layer-frames.html and then maybe come back with some questions ;)

Reply:
Oh, I think this doc needs some updates...